### ISSUE TITLE
localhost not responding

### ISSUE THREAD

**Issue by rmfranken** [RMLio/RMLStreamer#48]
Trying to run the docker version of the streamer since I'm not tech-savvy enough to set the whole thing up myself. Followed the instructions (I think), ran docker compose up, which looks like it does stuff as it starts a bunch of services. (docker volume inspect docker_data also worked) However, by the end when I try to access my http://localhost:8081 I get no response. I attached the terminal output below. Any tips on what I'm missing? I find the error to be quite cryptic.

[errors.txt](https://github.com/RMLio/RMLStreamer/files/11866402/errors.txt)

**Comment by s-minoo** [comment]
Did you perhaps pressed `Ctrl + C`  after running RMLStreamer's docker container?

**Comment by rmfranken** [comment]
Yes, as I put in the txt file in a comment (maybe I should have been more explicit in my issue), I ran the commands successfully, but after seeing that the localhost was not returning anything i crl+c'd out. Which produced more errors in my txt. Not sure if those are helpful for solving the underlying problem.
